initfiles/bin/init_thorslave.in

```diff
@@ -102,7 +102,8 @@ start_slaves()
         rsync_att=3
         rsync_stat=1
         while [[ $rsync_stat -ne 0 && $rsync_att -gt 0 ]] ; do
-            rsync -e "ssh -o LogLevel=QUIET -o StrictHostKeyChecking=no" --timeout=60 $master:$instancedir/slaves $slavesfname
+            # reset LD_LIBRARY_PATH here so ssh cmd doesn't use HPCC libssl/crypto as they may be different
+            LD_LIBRARY_PATH=: rsync -e "ssh -o LogLevel=QUIET -o StrictHostKeyChecking=no" --timeout=60 $master:$instancedir/slaves $slavesfname
```

Comment:
is this always okay. e.g. if something else has set LD_LIBRARY_PATH up system wide before Thor starts.
Should hpcc_setenv preserve the state on startup and set it to that here instead of blank?

Reply:
I think its ok because we are running ssh and that always starts a new shell and does not carry over LD_LIBRARY_PATH env to the remote machine.
If we really want new shell to have specific env vars we can put them in remote machine .bashrc or in ssh config or ssh cmdline provided to frunssh (sendEnv)